pkg/resource/logging-credentials/logging_operator_secrets.go

```diff
@@ -130,6 +151,34 @@ func AddLoggingCredentials(cluster *capi.Cluster, loggingCredentials *v1.Secret)
 	return secretUpdated, nil
 }
 
+// AddTracingCredentials - Add credentials to TracingCredentials secret if needed
+func AddTracingCredentials(cluster *capi.Cluster, tracingCredentials *v1.Secret) (bool, error) {
```

Comment:
@QuantumEnigmaa I think the AddTracingCredentials can be moved to a separate function (as it should be the same as the one for logs) right?